packages/devtools_extensions/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.0.3-wip
```

Comment:
Why not dev? What is the difference?

Reply:
I believe wip indicates a package that is not yet published, but I don't think it is enforced. 
@jonasfj are there official pub guidelines around how you should version an unpublished package?

Reply:
Not from pub, there are no guidelines.

For external package maintenance the Dart team generally follows:
https://github.com/dart-lang/sdk/wiki/External-Package-Maintenance#publishing-a-package

I didn't write it, and I'll admit I sometimes fail to follow it, but I like the idea of using a `-wip` to denote the fact that the version haven't been published yet.